packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -1625,12 +1625,12 @@ abstract class RenderObjectElement extends BuildableElement {
   void update(RenderObjectWidget newWidget) {
     super.update(newWidget);
     assert(widget == newWidget);
-    assert(() { debugUpdateRenderObjectOwner(); return true; });
+    assert(() { debugUpdatePipelineManager(); return true; });
     widget.updateRenderObject(this, renderObject);
     _dirty = false;
   }
 
-  void debugUpdateRenderObjectOwner() {
+  void debugUpdatePipelineManager() {
     _renderObject.debugOwner = debugGetOwnershipChain(10);
```

Comment:
Oh, this is actually a different thing.  We should probably rename this concept to avoid collisions with "pipeline owner".  This is just meant to be debug information about who created the render object.  Maybe `debugCreator` would be a better name.

Feel free to do the rename in a separate patch.  However, we shouldn't rename debugUpdateRenderObjectOwner to debugUpdatePipelineManager in this patch.  Either leave it alone or rename it to something like `debugUpdateRenderObjectCreator`